File: src/test/resources/output/primos.cs
// Código generado desde EBDA
using System;

class Program {
    static void Main(string[] args) {
        int n = 20;
        int i = 2;
        bool buscando = true;
        Console.WriteLine("Numeros primos hasta 20:");
        while (buscando) {
            int divisor = 2;
            int esPrimo = 1;
            bool revisando = true;
            while (revisando) {
                int modulo = i / divisor;
                modulo = modulo * divisor;
                if (modulo == i) {
                    esPrimo = 0;
                }
                divisor++;
                revisando = divisor < i;
            }
            if (esPrimo == 1) {
                Console.WriteLine(i);
            }
            i++;
            buscando = i < n;
        }
        return;
    }
}

File: src/test/resources/output/primos.py
# Código generado desde EBDA

n = 20
i = 2
buscando = True
print("Numeros primos hasta 20:")
while buscando:
    divisor = 2
    esPrimo = 1
    revisando = True
    while revisando:
        modulo = i / divisor
        modulo = modulo * divisor
        if modulo == i:
            esPrimo = 0
        divisor += 1
        revisando = divisor < i
    if esPrimo == 1:
        print(i)
    i += 1
    buscando = i < n
print('Resultado final: ' + str(i))
